### PR TITLE
refactored column ensemble forecaster

### PR DESCRIPTION
**Pull request by Aparna-Sakshi**
#### Reference Issues/PRs
Issue: #1678 

#### What does this implement/fix? Explain your changes.
Refactored Column Ensemble Forecaster by adding get_test_params method

#### Does your contribution introduce a new dependency? 
No

### CODE REVIEW

**Review by GuzalBulatova — COMMENTED** [review]
This looks good to me, well done, Aparna!
To me it seems that test parameters should belong in tests, not in the class itself though. Is this a part of some big refactoring?

**Comment by Aparna-Sakshi** [comment]
> This looks good to me, well done, Aparna! To me it seems that test parameters should belong in tests, not in the class itself though. Is this a part of some big refactoring?

Yes, the `get_test_params` function returns the test parameters. The idea is to decentralize the estimator parameters to the class itself. At present all the test parameters are being entered in `tests/_config` which could be nontrivial for a new user. This refactor could also reduce the chances of multiple people editing the same `tests/_config` file.

**Comment by Aparna-Sakshi** [comment]
@fkiraly  @GuzalBulatova, Could you please review this PR and let me know if any additional changes are required. I haven't yet removed the column ensemble test parameters from `tests/_config` in this PR, but I have commented out that line in my local system and checked if the tests works or not and it works.

File: sktime/forecasting/compose/_column_ensemble.py [hunk 1]
```diff
@@ -221,3 +221,22 @@ def _check_forecasters(self, y):
...
             )
         return self.forecasters
+
+    @classmethod
+    def get_test_params(cls):
+        """Return testing parameter settings for the estimator.
+
+        Returns
+        -------
+        params : dict or list of dict, default = {}
```
Comment (GuzalBulatova) [inline sktime/forecasting/compose/_column_ensemble.py:231]: `default={}`, without spaces

File: sktime/forecasting/compose/_column_ensemble.py [hunk 2]
```diff
@@ -221,3 +221,22 @@ def _check_forecasters(self, y):
...
+            Parameters to create testing instances of the class
```
Comment (GuzalBulatova) [inline sktime/forecasting/compose/_column_ensemble.py:232]: Please add a dot here and at the end of the line 235

**Review by GuzalBulatova — CHANGES_REQUESTED** [review]
Looks all good to me, have only tiny comments regarding the docstring. Please remove the test parameters from config to ensure that the tests run, and then I think it'll be ready to merge.

File: sktime/forecasting/compose/_column_ensemble.py (hunk 1 above)
Comment (Aparna-Sakshi) [reply sktime/forecasting/compose/_column_ensemble.py]: Done! Thanks for the feedback.

File: sktime/forecasting/compose/_column_ensemble.py (hunk 2 above)
Comment (Aparna-Sakshi) [reply sktime/forecasting/compose/_column_ensemble.py]: Done! Thanks for the feedback.

**Review by fkiraly — APPROVED** [review]
Yes, looks good, thanks!
@GuzalBulatova's comments seem addressed to me, do you want to quickly check, @GuzalBulatova?

**Review by GuzalBulatova — APPROVED** [review]
Looks ready to merge to me, good job, Aparna!

**Pull request merged**